server/controllers/services/email.js

```diff
@@ -207,7 +220,30 @@ module.exports = (app) => {
         return Promise.map(users, (user) => emailLib.send('email.approve', `members@${slug}.opencollective.com`, getData(user), {bcc:user.email}));
       })
       .then(() => res.send('Mailgun webhook processed successfully'))
-      .catch(next);
+      .catch(e => {
+        switch (e.message) {
+          case 'no_subscribers':
```

Comment:
But they are used twice, e.g lines 194 + 225. If you change one without the other, code breaks.